hal/src/main/native/athena/DIO.cpp

```diff
@@ -114,6 +122,8 @@ HAL_DigitalHandle HAL_InitializeDIOPort(HAL_PortHandle portHandle,
   }
 
   digitalSystem->writeOutputEnable(outputEnable, status);
+  port->previousAllocation =
+      allocationLocation == nullptr ? "" : allocationLocation;
```

Comment:
I tend to prefer the construct `allocationLocation ? allocationLocation : “”`

Reply:
Fixed